OBSDOCS-462: Update logging uninstall docs (#69255)
openshift/openshift-docs: modules/uninstall-cluster-logging-operator.adoc

// Module included in the following assemblies:
//
// * logging/cluster-logging-uninstall.adoc

:_mod-docs-content-type: PROCEDURE
[id="uninstall-cluster-logging-operator_{context}"]
= Uninstalling the {logging}

> You can stop log aggregation by deleting the {clo} and the `ClusterLogging` custom resource (CR).

Reviewer: ```suggestion
You can stop aggregating logs by deleting the {clo} and the `ClusterLogging` custom resource (CR).
```
- Maybe better to verb-ify this phrase? Just a suggestion.
- Is "aggregate" the right word? Or should it be "collect"? In my mind, they mean two slightly different things.

Author: I'd say aggregate is right; collection is specifically done by the log collector I think.

.Prerequisites

* You have administrator permissions.
* You have access to the *Administrator* perspective of the {product-title} web console.

.Procedure

> . Navigate to the *Administration* -> *Custom Resource Definitions* page. Click *ClusterLogging*.

Reviewer: ```suggestion
. Go to the *Administration* -> *Custom Resource Definitions* page, and click *ClusterLogging*.
```
- [The ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n) says to avoid using "navigate" and to use alternatives such as "go to" when you can.
- I would combine these two sentences into one because these two actions are presented as a single step. If you keep them as separate sentences, it feels to me like they should be presented as separate steps.

. On the *Custom Resource Definition Details* page, click *Instances*.

> . Click the options menu {kebab} next to the instance, then click *Delete ClusterLogging*.

Reviewer: ```suggestion
. Click the options menu {kebab} next to the instance, and click *Delete ClusterLogging*.
```
Edit to make the wording consistent with the other options menu step that follows.

. Navigate to the *Administration* -> *Custom Resource Definitions* page.

> . Click the options menu {kebab} next to *ClusterLogging* and select *Delete Custom Resource Definition*.

Reviewer: ```suggestion
. Click the options menu {kebab} next to *ClusterLogging*, and select *Delete Custom Resource Definition*.
```
Technically, these are two independent clauses joined by a coordinating conjunction, so a comma is needed before `and`.

+

> [NOTE]

Reviewer: If data loss can occur, I'd recommend changing this to a Warning admonition instead of a Note: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

====
Deleting the `ClusterLogging` CR does not remove the persistent volume claims (PVCs). To delete the remaining PVCs, persistent volumes (PVs), and associated data, you must take further action. Releasing or deleting PVCs can delete PVs and cause data loss.
====

> . If you have created a `ClusterLogForwarder` CR, click the options menu {kebab} next to *ClusterLogForwarder*, then click *Delete Custom Resource Definition*.

Reviewer 2: This part isn't consistent with the rest of the document in terms of deleting the instance first, then the CRD (like it is mentioned for `LokiStack` `ClusterLogging` and `Elasticsearch` instances).
It is technically correct though, deleting the CRD will delete its instances. I am only mentioning it for the sake of consistency.

> . If you have created a `ClusterLogForwarder` CR, click the options menu {kebab} next to *ClusterLogForwarder*, then click *Delete Custom Resource Definition*.

Reviewer: ```suggestion
. If you have created a `ClusterLogForwarder` CR, click the options menu {kebab} next to *ClusterLogForwarder*, and then click *Delete Custom Resource Definition*.
```
- These are two independent clauses, so they need to be joined with a coordinating conjunction after the comma.

> . Navigate to the *Operators* -> *Installed Operators* page.

Reviewer: ```suggestion
. Go to the *Operators* -> *Installed Operators* page.
```

> . Click the options menu {kebab} next to the {clo}, then click *Uninstall Operator*.

Reviewer: ```suggestion
. Click the options menu {kebab} next to the {clo}, and then click *Uninstall Operator*.
```

. Optional: Delete the `openshift-logging` project.
+

> [IMPORTANT]

Reviewer: Again, if potential data loss is in the picture, I'd recommend using a WARNING admonition instead.

====
Deleting the `openshift-logging` project deletes everything in that namespace, including any persistent volume claims (PVCs). If you want to preserve logging data, do not delete the `openshift-logging` project.
====

> .. Navigate to *Home* -> *Projects*.

Reviewer: ```suggestion
.. Go to *Home* -> *Projects*.
```

> .. Click the options menu {kebab} next to the *openshift-logging* project, then click *Delete Project*.

Reviewer: ```suggestion
.. Click the options menu {kebab} next to the *openshift-logging* project, and then click *Delete Project*.
```
